src/utils/geteditornamespace.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @license Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import loadScript from 'load-script';
+
+/* global window CKEDITOR */
+
+let promise, mock;
+
+export function mockMethod( callback ) {
```

Comment:
Oh no, we cannot do that :smile: If you really need to hack into method internals, make them public. So instead of making `mock` object available, do it with `loadScript` library. You could just:

```javascript
getEditorNamespace.scriptLoader = loadScript;
```

making this property open for mocking. Also, I recommend promising `loadScript` before it's even attached as a static member for easier mocking later.